src/backends/perf_pt/mod.rs

```diff
@@ -58,6 +57,55 @@ extern "C" {
     fn perf_pt_free_tracer(tr_ctx: *const c_void) -> c_int;
 }
 
+/// A Intel PT trace, obtained via Linux Perf.
```

Comment:
"A*n* Intel". Also is "perf" capitalised or not (I'm not sure)?

Reply:
Yeah, you are right. Lower case in the kernel docs.